### PR TITLE
Support for React 15

### LINKED ISSUE #389: Support React 15
Upgrade fluxible-addons-react and fluxible-router to support React 15. I will release an RC version of each before React 15 is fully released.

### PR DESCRIPTION
**Pull request by mridgway**
Resolves #389.

### CODE REVIEW

**Comment by yahoocla** [comment]
Thank you for submitting this pull request, however I do not see a valid CLA on file for you. Before we can merge this request please visit https://yahoocla.herokuapp.com/ and agree to the terms. Thanks! :smile:

**Comment by yahoocla** [comment]
CLA is valid!

File: package.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
     "chai": "^3.2.0",
     "coveralls": "^2.11.1",
     "es6-promise": "^3.1.2",
-    "eslint": "^2.2.0",
+    "eslint": "~2.2.0",
```
Comment (redonkulus) [inline package.json]: ^

**Comment by redonkulus** [comment]
:+1:

**Comment by lingyan** [comment]
:+1: :ship: :fireworks:

**Comment by redonkulus** [comment]
squash first

**Pull request merged**